en/orm/entities.rst

```diff
@@ -91,7 +91,13 @@ will access the data in an entity using object notation::
     $article->title = 'This is my first post';
     echo $article->title;
 
-You can also use the ``get()`` and ``set()`` methods::
+You can also use the ``get()`` and ``set()`` methods.
+
+.. php:method:: set($field = null, $value = null)
```

Comment:
Are you sure the field can be nullable? does it make any sense to just call set() without arguments?

Reply:
You're right, the field can't be null. I just moved this piece from one place to another and obviously missed out on this detail. It should actually be:

```
.. php:method:: set($field, $value = null, array $options = [])
```

Not sure if it makes sense to add the `$options` array here too?